text/0000-fabric-private-chaincode.md

```diff
@@ -3,279 +3,282 @@ layout: default
+ 
+ 
 Terminology:
 
-* Trusted Execution Environment (TEE): The isolated secure environment in which programs run in encrypted memory, unreadable even by privileged users or system processes. FPC chaincodes run in TEEs.
-
-* Enclave: The Intel SGX-specific name for a Trusted Execution Environment (TEE). In the initial release of FPC, the TEE and Enclave terms can be used interchangeably, as the Intel SGX Enclave is the first TEE supported because to date, it is the only TEE that supports remote attestation as required by the FPC integrity architecture.
-
-* Attestation: An important cryptographic feature of Trusted Execution Environments by which the hardware can reliably state exactly what software it is running. This statement is signed by the hardware so that anyone reading it can verify that the statement came from an actual valid and up-to-date TEE. The Attestation can cover both the software portion of the TEE itself as well as any program to be run in it. This attestation takes the form of a "quote" containing (among other things) a measurement of the code and data in the enclave and its software version number, which is signed by the TEE and can be verified by anyone using well known Public Key Infrastructure technology.
-
-Note on Terminology: The current feature naming scheme includes several elements that are Intel SGX-specific; for example, the term "Enclave" as a specific type of Trusted Execution Environment. We intend to revise the naming to reflect that we intend to support other TEEs such as AMD SEV-SNP, etc. The project team currently participates in the new Confidential Computing Consortium which aims to promote standards for deployment of workloads across TEEs, and we intend to align with their terminology.
-
-
+* Attestation: An important cryptographic feature of Trusted Execution Environments by which the hardware can reliably state exactly what software it is running. This statement is signed by the hardware so that anyone reading it can verify that the statement came from an actual valid and up-to-date TEE. The Attestation can cover both the software portion of the TEE itself as well as any program to be run in it. This attestation takes the form of a "quote" containing (among other things) a measurement of the code and data in the TEE and its software version number, which is signed by the TEE and can be verified by anyone using well known Public Key Infrastructure technology. 
+ 
+* Trusted Execution Environment (TEE): The isolated secure environment in which programs run in encrypted memory, unreadable even by privileged users or system processes. FPC chaincodes run in TEEs. 
+ 
```

Comment:
TEEs should appear before Attestation
```suggestion
* Trusted Execution Environment (TEE): The isolated secure environment in which programs run in encrypted memory, unreadable even by privileged users or system processes. FPC chaincodes run in TEEs. 

* Attestation: An important cryptographic feature of Trusted Execution Environments by which the hardware can reliably state exactly what software it is running. This statement is signed by the hardware so that anyone reading it can verify that the statement came from an actual valid and up-to-date TEE. The Attestation can cover both the software portion of the TEE itself as well as any program to be run in it. This attestation takes the form of a "quote" containing (among other things) a measurement of the code and data in the TEE and its software version number, which is signed by the TEE and can be verified by anyone using well known Public Key Infrastructure technology. 

```